tbk/onepay/sign.py

```diff
@@ -0,0 +1,49 @@
+# encoding: utf-8
+from __future__ import unicode_literals
+
+import hashlib, hmac, base64
+from tbk import onepay
+
+def sign_sha256(secret: str, data: str) -> str:
+    digest = hmac.new(str.encode(secret), msg=str.encode(data), digestmod=hashlib.sha256).digest()
+    return base64.b64encode(digest).decode()
+
+class SignUtil(object):
+
+    @classmethod
+    def build_signature_transaction_create_request(cls, signable, secret: str):
+        external_unique_number_as_string = str(signable.external_unique_number)
+        total_as_string = str(signable.total)
+        items_quantity_as_string = str(signable.items_quantity)
+        issued_at_as_string = str(signable.issued_at)
+
+        data = str(len(external_unique_number_as_string.encode('utf-8'))) + external_unique_number_as_string
```

Comment:
This seems too repetitive for my python aesthetic sense :). I would create a method to do the concatenation work of the signing, like:

```
def concat_for_signing(*params):
    return ''.join(str_with_len_prefix(param) for param in params)

def str_with_len_prefix(param):
  return str(len(str(param).encode('utf-8'))) + str(param)
```

then you can rewrite all these lines with the simpler and clearer:
```
data = concat_for_signing(
    signable.external_unique_number, signable.total, signable.items_quantity, signable.issued_at, 
    onepay.callback_url)
```

Even better: if the signable class itself (`TrasactionCreateRequest`) had the responsibility to say which fields should be signed, you could do something like:

``` 
data = concat_for_signing(*signable.signable_data)
```

And the method would be waay more readable.

cc: @eugenioLopezRamos who might have a similar problem on the Ruby SDK

Reply:
(And if the argument for not having `signable_data` on `TrasactionCreateRequest` is that there is one piece of data to sign which is not on the request, that's easy to fix by having `signable_data` to receive the callback_url as a parameter)

Reply:
Also: If this issue is present in PHP and Java and .NET (I surely missed it there) can you guys create issues on those project to refactor them in a similar way (i.e, that the signable class should be responsible for saying which fields are part of the signature)